### ISSUE TITLE
Weight decay is applied to the rating embeddings, biases and normalization gains

### ISSUE THREAD

**Issue by mmmfrieddough** [mmmfrieddough/anthro-chess#566]
## Symptom

Weight decay is applied to every parameter the model owns, including the rating
conditioning embeddings, all biases and all normalization gains. Turning decay
on therefore suppresses the rating signal far harder than it suppresses the
weights it is meant to bound.

Measured on two arms that differ only in `weight_decay`, at matched step
347,200, parameter L2 norm by group:

| group | control | `weight_decay = 0.01` | ratio |
| --- | ---: | ---: | ---: |
| `rating_embedding` | 37.662 | 5.433 | **0.14x** |
| biases | 269.637 | 35.910 | **0.13x** |
| other embeddings | 22.423 | 4.498 | 0.20x |
| `action_head` | 158.661 | 48.256 | 0.30x |
| normalization gains | 31.477 | 17.630 | 0.56x |
| whole model | 1652.464 | 418.533 | 0.25x |

The rating embedding is driven to 14% of the control while the model overall
sits at 25%, so the conditioning signal is hit about twice as hard as the
matrices the decay was aimed at. There are only three such vectors, weak,
strong and unrated, so they receive thin gradient signal and the uniform decay
pull dominates them.

## What it costs, measured

The same pair, at the same matched step, on the benchmarks that read rating
conditioning:

- `dependency.rating_absent_degradation` 0.15281 to 0.10865, **28.9% worse**.
  Removing the rating input hurts the decayed arm less, because it is barely
  using it.
- `dependency.rating_anchor_policy_divergence` 27.0% worse,
  `rating_shuffled_degradation` 18.8% worse,
  `rating_cross_conditioning_penalty` 16.7% worse.
- `ladder.rating_ladder_error` worse at every temperature including greedy,
  by 6.9% to 17.4%. A model whose conditioning is suppressed cannot hit a
  configured rating.

Rating-conditioned play is the product. This is the most expensive way for the
setting to go wrong and it fails silently, since nothing errors and the
held-out loss actually improves.

## Cause

`src/anthro_chess/training/runner.py`:

```python
optimizer = torch.optim.AdamW(
    model.parameters(),
    lr=config.learning_rate,
    betas=(_FIRST_MOMENT_DECAY, config.second_moment_decay),
    weight_decay=config.weight_decay,
    fused=_fused_optimizer(device),
)
```

One parameter group, so the coefficient reaches everything. The usual recipe
decays matrix parameters and exempts the rest, because parameters with small or
sparse gradients have nothing to push back against a uniform pull.

There is no decision record or comment arguing for the single group. It was
never wrong before: `weight_decay` has been 0.0 in every configuration this
repository ships, and at zero the grouping cannot matter. The defect is dormant
rather than active, and the first run to turn decay on is the one that pays.

## Scope

- Split the optimizer into a decayed group and an exempt group, decaying
  parameters of two or more dimensions and exempting the rest. That exempts
  exactly the measured casualties: the rating embeddings, every bias and every
  normalization gain are all one-dimensional.
- The change is inert for every run recorded so far. At `weight_decay = 0.0`
  two groups at zero behave identically to one, so no stored reading moves and
  `training_sha256` is untouched, the optimizer construction being outside it.
  The pull request should demonstrate that rather than assert it.

## Acceptance Criteria

- A test asserts the exempt group holds the rating embeddings, the biases and
  the normalization gains, and that the decayed group holds the weight matrices.
- A test asserts that at `weight_decay = 0.0` the grouping changes no step, by
  running two short arms under strict determinism on either side of the change
  and showing their readings agree exactly. This is the cheap demonstration
  `docs/issue-workflow.md` asks for from a change meant to leave the weights
  alone.
- The frozen vehicle digest is unchanged, asserted by the existing test.
- One arm at `model_dim` 128 and `weight_decay = 0.01` with the fix applied,
  trained to at least 347,200 steps, reports `rating_embedding` norm and the
  four `dependency.*` metrics against the arms recorded for `#562`. The reading
  states whether the conditioning damage is removed.

## Required Environment

The implementation, the grouping test and the inertness demonstration need no
GPU. The acceptance arm needs one CUDA host with the widened corpus prepared,
roughly 30 GPU-hours.

## References

- `#562`, where this surfaced, and the arms `h562-wd-w128-0p01` and
  `h490-trunk-0p003` the table above is read from
- `docs/decisions/0089-bounded-growth-removes-the-ceiling-and-costs-a-flat-offset.md`
- `docs/decisions/0008-rating-temperature-independence.md`, for why the
  conditioning signal is not substitutable

**Comment by mmmfrieddough** [comment]
Claimed for implementation by Claude Code (Opus 5).

Session: 34f68598-084e-4363-9388-f23825bb7051
